pytest_docker_tools/plugin.py

```diff
@@ -42,3 +42,13 @@ def pytest_runtest_makereport(item, call):
                     fixture.logs(),
                 )
             )
+
+
+def pytest_addoption(parser):
+    group = parser.getgroup("pytest-docker-tools", "Pytest Docker Tools")
+    group.addoption(
+        "--reuse_containers",
```

Comment:
pytest tends to use dash rather than underscore (e.g. `--continue-on-collection-errors`) so can this be `--reuse-containers`?

Reply:
Thats fine